### DISCUSSION TITLE
how do i make 2 frames with a gap??

### DISCUSSION THREAD

**Discussion by scarlett-pixel** [keiwando/toonsquid#641] [Q&A]
![IMG_4321](https://github.com/user-attachments/assets/97550ad9-b5c4-42e2-8368-db83fd1d8187)

this happens 2 me alot and it makes me angry💔 basically when i make a frame, then within a gap i make another frame and the first frame automatically extends to reach the other frame
the problem is that i don’t want this to happen i just want frames with a gap within eachother

**Comment by SharpsterDraws** [comment]
You have the magnet icon turned on… the bottom icon on the left 🧲.  Turn that off and you can have gaps in your frames.
